modules/library/referencing/src/main/java/org/geotools/referencing/wkt/Symbols.java

```diff
@@ -104,7 +104,7 @@ public class Symbols {
      * Consequently, each instances of {@link Parser} or {@link Formatter} must use a clone of this
      * object, not this object directly (unless they synchronize on it).
      */
-    final NumberFormat numberFormat;
+    public final NumberFormat numberFormat;
```

Comment:
Don't think you're actually using this, but generally speaking, not a good idea (to make a field public, outside of cases where the class is basically a Struct) especially for a non thread safe field.

Reply:
Correct, it was there for an initial phase of the development, but then I have removed the Symbols import. Fixing it